verification/cbmc/proofs/aws_cryptosdk_hdr_write/Makefile

```diff
@@ -25,7 +25,7 @@ HARNESS_FILE = $(HARNESS_ENTRY).c
 # Values are chosen for performance. 
 # Increaseing either does not increase the coverage. 
 MAX_EDK_LIST_ITEMS ?= 1
-MAX_IV_LEN ?= 12
+MAX_IV_LEN ?= 255 # Max value for uint8
```

Comment:
Why do we need this bound at all? I don't see it in any loop unwinding.

Reply:
It's to avoid an arithmetic overflow from a cast, `(uint8_t)hdr->iv.len)`